question-service/app/crud/questions.py

```diff
@@ -39,7 +39,11 @@ async def delete_question(question_id: str):
 async def update_question_by_id(question_id: str, question_data: UpdateQuestionModel):
     existing_question = await question_collection.find_one({"_id": ObjectId(question_id)})
     if existing_question is None:
-        return None
+        return {"error": "does not exist"}
```

Comment:
Might be better to do this using exception handling, maybe a new issue to convert all of this to use error handling